src/components/createRecipe/recipeForm/Step2.jsx

```diff
@@ -49,7 +49,6 @@ function Step2(props) {
       // Then
       setImgUrl([imageUrl.data.secure_url]);
     } catch (error) {
-      console.log(error);
```

Comment:
Can you add here a setState so that if you get an error you can load an error message on the screen. The goal was to remove all the console logs because they are not helpful for the user. But removing every kind of notification to the user that something happened is not the goal.  Hope it makes sense.

Reply:
Good catch!

Reply:
yeah !!! makes sense!